swath_projector/earthdata_varinfo_config.json

```diff
@@ -28,6 +28,29 @@
         "/support_data/step_2_N_value_residual",
         "/support_data/temp_sensitivity_ratio"
       ]
+    },
+    {
+      "Applicability": {
+        "Mission": "TEMPO",
+        "ShortNamePath": "TEMPO_NO2_L2"
+      },
+      "VariablePattern": [
+        "/support_data/gas_profile",
+        "/support_data/scattering_weights",
+        "/support_data/scattering_weights_clear_sky",
+        "/support_data/temperature_profile"
+      ]
+    },
+    {
+      "Applicability": {
+        "Mission": "TEMPO",
+        "ShortNamePath": "TEMPO_NO2_L2_NRT"
+      },
+      "VariablePattern": [
+        "/support_data/gas_profile",
+        "/support_data/scattering_weights",
+        "/support_data/temperature_profile"
```

Comment:
The excluded variables are all (I think) just due to higher dimensionality. For example, `/support_data/temperature_profile` has dimensions: `(mirror_step, xtrack, swt_level)`.

Without something like DAS-943 being completed, to cope with higher dimensionality and, in particular, having non-spatial dimensions last, we'll need to keep adding collection-specific excluded science variables for each product.